deps/anvil/madara_state_update.sh

```diff
@@ -1,11 +1,18 @@
 #!/bin/bash
 
-RPC_URL="http://madara:9945"
-BLOCK_NUMBER="0"
-
 # Default Anvil private key
 PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
 ANVIL_URL="http://anvil:8545"
+RPC_URL="http://madara:9945"
+
+BLOCK_NUMBER=$(( $(curl --silent --location "$RPC_URL"/v0_7_1/ \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "jsonrpc": "2.0",
+    "method": "starknet_blockNumber",
+    "params": [],
+    "id": 1
+  }' | jq -r '.result') + 1 ))
```

Comment:
Maybe catch errors here if curl fails for whatever reason

Reply:
Done!